### PR TITLE
Add release github action

### PR DESCRIPTION
**Pull request by Sheikah45**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    container: adoptopenjdk:11-jdk-hotspot-bionic
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 15
+        uses: actions/setup-java@v1
+        with:
+          java-version: 15.0.1
+          java-package: jdk+fx
```
Comment (Brutus5000) [inline .github/workflows/release.yml]: JDK15 collides with the container description above using JDK11. does that work?

Comment (Sheikah45) [reply .github/workflows/release.yml]: Nope this is just a victim of copy and paste nad not being able to test actions before they are on develop

Comment (Sheikah45) [reply .github/workflows/release.yml]: should be better now

**Review by Brutus5000 — APPROVED** [review]
Let's see if it actually works :)

**Pull request merged**